pkg/aws/cloud.go

```diff
@@ -128,3 +129,18 @@ func (c *defaultCloud) IsArnManaged(arn string) (bool, error) {
 func getManagedByTag(cfg CloudConfig) string {
 	return fmt.Sprintf("%s/%s/%s", cfg.AccountId, cfg.ClusterName, cfg.VpcId)
 }
+
+func IgnoreNotFound(err error) error {
```

Comment:
Unit tests would be nice for these new utilities

Reply:
ok

Reply:
removed